src/samplers/ratelimiting_sampler.js

```diff
@@ -32,7 +32,7 @@ export default class RateLimitingSampler {
         }
 
         this._maxTracesPerSecond = maxTracesPerSecond;
-        this._rateLimiter = new RateLimiter(maxTracesPerSecond);
+        this._rateLimiter = new RateLimiter(maxTracesPerSecond, 2.0);
```

Comment:
I am not convinced. Currently [this test](https://github.com/uber/jaeger-client-node/blob/master/test/samplers/ratelimiting_sampler_test.js#L28) is incorrect because (a) initially it succeeds due to constructor bug(?) I pointed out, and after timer is advanced it succeeds because we're only doing a single check. If we change it to 9-10 checks, it will fail with your change.

Remember that RateLimitingSampler was originally designed for N > 1 traces per second. If you want it to work for N << 1, then let's test both scenarios.